charts/cluster-addons/templates/cni/calico.yaml

```diff
@@ -49,6 +49,92 @@ spec:
     - secret:
         name: {{ include "cluster-addons.componentName" (list . "cni-calico") }}-config
         key: overrides
+{{- if .Values.cni.calico.metadataGlobalNetworkPolicy }}
+---
+apiVersion: addons.stackhpc.com/v1alpha1
+kind: Manifests
+metadata:
+  name: {{ include "cluster-addons.componentName" (list . "cni-calico-metadataGlobalNetPol") }}
+  labels: {{ include "cluster-addons.componentLabels" (list . "cni-calico-metadataGlobalNetPol") | nindent 4 }}
+  annotations:
+    # Tell Argo to ignore the non-controller owner references for this object
+    argocd.argoproj.io/sync-options: "ControllerReferencesOnly=true"
+spec:
+  clusterName: {{ include "cluster-addons.clusterName" . }}
+  bootstrap: true
+  targetNamespace: {{ .Values.cni.calico.release.namespace }}
+  releaseName: cni-calico-metadata-global-netpol
+  manifestSources:
+    # calico-kube-controllers
+    - template: |
+        ---
+        apiVersion: projectcalico.org/v3
+        kind: GlobalNetworkPolicy
+        metadata:
+          name: {{ include "cluster-addons.componentName" (list . "deny-metadata-egress") }}
+        spec:
+          order: {{ .Values.cni.calico.metadataGlobalNetworkPolicy.metadataDenyPriority }}
+          types:
+            - Egress
+          egress:
+            - action: Deny
+              destination:
+                nets:
+                  {{- range .Values.cni.calico.metadataGlobalNetworkPolicy.denyEgressCidrs }}
+                    - {{ . }}
```

Comment:
Reduce indent by two here (just for style) and other similar places

Reply:
I moved some lines over. I think  you meant the list below  `nets:` right?